elm.json

```diff
@@ -8,17 +8,23 @@
         "direct": {
             "elm/browser": "1.0.1",
             "elm/core": "1.0.2",
-            "elm/html": "1.0.0"
+            "elm/html": "1.0.0",
+            "elm/json": "1.1.3",
+            "elm-community/list-extra": "8.2.0",
+            "elm-explorations/test": "1.2.1",
```

Comment:
shouldn't be this in `test-dependencies` 🤔 ?

Reply:
It's an `elm install` bug, mostly. Some packages regularly depend on `elm-explorations/test` and then `elm make` flips over and refuses to do anything, and you have to remove `test` from the regular deps, and only have it in the test deps. Happens to our codebase almost every time somebody touches elm.json.